README.md

```diff
@@ -86,4 +86,29 @@ export class ExampleModule {
     return module;
   }
 }
-```
\ No newline at end of file
+```
+
+## Queuing mechanism
+
+```mermaid
```

Comment:
Add a chart describing how the queues flow looks like
[See here](https://github.com/across-protocol/scraper-api/tree/amatei/acx-949-update-scraper-api-to-handle-new-spoke#queuing-mechanism)